### PR TITLE
Initial assessment repo setup

### PR DESCRIPTION
**Pull request by Genyus**
Closes https://codingblackfemales.atlassian.net/browse/TECH-408

### CODE REVIEW

**Review by sohbaker — COMMENTED** [review]
Are we wanting students to select and install their own library for using Json with Java or making a recommendation? Either way, we'll probably need to add something to the README about this.

**Comment by Genyus** [comment]
They should be free to choose, but I'll add a couple of recommendations

File: README.md [hunk 1]
```diff
@@ -1,2 +1,65 @@
...
+#### **Learning Outcomes:**
+- Craft clean, efficient, and maintainable code.
+- Harness Java & SpringBoot to its fullest.
+
+---
+
+## **Deliverables / What You Should Submit:**
+1. **Repository Link**: A GitHub link showing off all your handiwork.
+1. **Documentation**: A top-tier README.md in your repository, going over setup, endpoints, and any other crucial bits and bobs.
+
```
Comment (efuaakum) [inline README.md:206]: Does this need a "Getting Started" section with initial steps on how to build and run the app?

Comment (Genyus) [reply README.md:206]: Good shout 👍🏾

**Comment by Genyus** [comment]
D'oh! Just realised this PR was issued directly against upstream. 🤦🏾‍♂️ Closing and will re-open in solutions repo

**Pull request closed without merging**